com.avaloq.tools.ddk.xtext.format/model/generated/Format.genmodel

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <genmodel:GenModel xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:ecore="http://www.eclipse.org/emf/2002/Ecore"
-    xmlns:genmodel="http://www.eclipse.org/emf/2002/GenModel" copyrightText="generated by Xtext 2.27.0.M3" modelDirectory="/com.avaloq.tools.ddk.xtext.format/src-gen"
+    xmlns:genmodel="http://www.eclipse.org/emf/2002/GenModel" copyrightText="generated by Xtext" modelDirectory="/com.avaloq.tools.ddk.xtext.format/src-gen"

```

Comment:
this changes are not correct. Could we revert the file?

Reply:
reverted.